README.md

```diff
@@ -2,6 +2,105 @@
 
 For architectural details, step-by-step instructions, and customization options, see the [deployment guide](https://aws-quickstart.github.io/quickstart-jfrog-artifactory/).
 
-To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo. 
+To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
 
 To submit code for this Quick Start, see the [AWS Quick Start Contributor's Kit](https://aws-quickstart.github.io/).
+
+## Upgrading RT and Xray Version
```

Comment:
Not sure if we want to include these here in a public repo. Sorry if I wasn't clear.

Reply:
Revert it?

Reply:
Yes, I think that's best.

Reply:
Done.